opendbc/car/honda/carstate.py

```diff
@@ -140,12 +144,20 @@ def update(self, cp, cp_cam, _, cp_body, __) -> structs.CarState:
                           cp.vl["DOORS_STATUS"]["DOOR_OPEN_RL"], cp.vl["DOORS_STATUS"]["DOOR_OPEN_RR"]])
     ret.seatbeltUnlatched = bool(cp.vl["SEATBELT_STATUS"]["SEATBELT_DRIVER_LAMP"] or not cp.vl["SEATBELT_STATUS"]["SEATBELT_DRIVER_LATCHED"])
 
+    # Triggered by STEER_TORQUE_REQUEST in CAN frame STEERING_CONTROL
+    self.steer_on = bool(cp.vl["STEER_STATUS"]["STEER_CONTROL_ACTIVE"])
```

Comment:
It's good that we're checking this, but looks like self.steer_on still needs to be added to your latching temp steer fault logic below. Maybe also rename to `self.eps_steer_active` or something else more descriptive?